packages/twenty-front/src/modules/ui/navigation/navigation-drawer/hooks/useFilteredWorkspaces.ts

```diff
@@ -0,0 +1,26 @@
+import { currentWorkspaceState } from '@/auth/states/currentWorkspaceState';
+import { useRecoilValue } from 'recoil';
+
+export const useFilteredWorkspaces = () => {
```

Comment:
Why relying on recoil to load currentWorkspaceState but on a parameter of searchWorkspace for the workspaces?

It makes this hooks hard to understand the dependencies of this hook from the outside and hide the fact that the currentWorkspace is hidden.

Also naming seems generic useFilteredWorkspaces but the folder suggest it's part of navigation